libs/common-hooks/src/index.ts

```diff
@@ -1,11 +1,9 @@
-export * from './useContract'
 export * from './useDebounce'
 export * from './usePrevious'
 export * from './useParsedQueryString'
 export * from './useIsMounted'
 export * from './useLoadingWithTimeout'
 export * from './useInterval'
-export * from './useNetworkName'
```

Comment:
Moved these 2 hooks out as they depend on other cowswap only modules.
Otherwise we'd need to pull in a lot of unrelated modules into explorer.